### ISSUE TITLE
How to install v7?

### ISSUE THREAD

**Issue by ricky11** [ali-sdk/ali-oss#929]
the v.7 branch shows to install use npm install ali-oss,  but this will install the master branch which is still v6.x

update the document  for 7.x branch to show how to install.

**Comment by eugle** [comment]
The higher the version, the larger the package. In 2021, this package is already 568kb

it's so funny

**Comment by ricky11** [comment]
You refer to 6.x right? the release notes have not been updated since version 6.8 so i hope they update the release note to show the changes.  I also notice the package size has gone up from 300kb to 500kb, actually i dont know why also.

**Comment by eugle** [comment]
@ricky11 It is very exaggerated to increase to 568kb. Although I used dynamic import to solve the initial bundling, it is still a problem. When used, 568kb of useless code will still be loaded, which is not a real solution.

**Comment by frenchvandal** [comment]
Is this project still active?

**Comment by ricky11** [comment]
Yes i also use dynamic import, but its still huge.. i hope v7 official support by core oss team will be launched soon, but more important they should update the release notes and the documentation for V7.  
there are now broken links in the docs for v7. and most of the doc is just copy/paste from v6.
https://github.com/ali-sdk/ali-oss/blob/ts/ts-guide.md

**Comment by ricky11** [comment]
any update from the team? @fengmk2
